components/ConnectPaypal.js

```diff
@@ -72,7 +72,7 @@ class ConnectPaypal extends React.Component {
       return (
         <P fontSize="Caption" color="black.500" mt={3}>
           <FormattedMessage
-            id="collective.connectedAccounts.paypal.connected"
```

Comment:
Not a blocker, but it's better to avoid changing the `id` of messages unless it's really necessary. By doing that, we're resetting the translations for the messages and contributors will have to translate them again in Crowdin.

Reply:
I agree this is far from ideal but I'm trying to improve consistency in our message ids.
I'll do it properly and rename the existing translations.